study.md

```diff
@@ -66,7 +68,9 @@ Which ActiveRecord method finds all of the records of a certain type (or
 entity)?
 
 ```md
-<!-- your response here -->
+The find_by method can be used to find all of the records of a certain type or entitity.  E.g.
```

Comment:
`#find_by` retrieves at most one record.  You would want to use `#all` here since we just want every record of a certain type, regardless of their attributes.